@ecomplus/storefront-components/src/js/SearchEngine.js

```diff
@@ -405,6 +405,11 @@ export default {
         }
       }
       this.fetchItems()
+    },
+    setSortOrder (sort) {
+      this.selectedSortOption = sort
+      this.ecomSearch.setSortOrder(sort)
+      this.fetchItems()
```

Comment:
```suggestion
      this.scheduleFetch()
```

funcionaria, mas acho que seria melhor inserirmos na fila com o método [`scheduleFetch`](https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-components/src/js/SearchEngine.js#L312-L322) pra evitar requisições excedentes.